### ISSUE TITLE
Keybindings out of sync with locale in PGP settings

### ISSUE THREAD

**Issue by Natureshadow** [neomutt/neomutt#2355]
## Expected Behaviour

When composing a message, then hit `p` to configure PGP for this message, I get the following prompt:

`PGP (s)ign, sign (a)s, PGP/M(i)ME format, (c)lear, or (o)ppenc mode off?`

I expect the letters in parantheses to do what they state.

## Actual Behaviour

I have seen some variations of this bug, depending on the locale and maybe other factors I could not yet identify. RIght now, none of the key bindings work expect for `(s)ign`. The key binding for `clear`, which I often use for mailing lists with people who would otherwise ask what this signature.asc thing is and why MS Word tells them it cannot open it, seems to be `b` right now.

I am running under the `nb_NO` locale, and my guess is that `clear` is `borte` under that locale, but I do not quite understand why the prompt itself is in English with mutt apparently being aware of my locale.

## Steps to Reproduce

Set the locale like this:

```
LANG=nb_NO.UTF-8
LANGUAGE=nb:nn:sv:en_GB:en_US:de
LC_CTYPE=nb_NO.UTF-8
LC_NUMERIC=nb_NO.UTF-8
LC_TIME=nb_NO.UTF-8
LC_COLLATE=C
LC_MONETARY=de_DE.UTF-8
LC_MESSAGES=nb_NO.UTF-8
LC_PAPER=de_DE.UTF-8
LC_NAME=nb_NO.UTF-8
LC_ADDRESS=de_DE.UTF-8
LC_TELEPHONE=nb_NO.UTF-8
LC_MEASUREMENT=nb_NO.UTF-8
LC_IDENTIFICATION=nb_NO.UTF-8
LC_ALL=
```

Start neomutt, hit m to compose a message, get to the compose screen. Hit `p` for PGP, then test the key bindings.

## How often does this happen?

- Always

## When did it start to happen?

It comes and goes and it sometimes has `y` for the `clear` option instead of `b`, when I was running under a subtly different locale perhaps.

## NeoMutt Version

```
NeoMutt 20191207
Copyright (C) 1996-2016 Michael R. Elkins and others.
NeoMutt comes with ABSOLUTELY NO WARRANTY; for details type 'neomutt -vv'.
NeoMutt is free software, and you are welcome to redistribute it
under certain conditions; type 'neomutt -vv' for details.

System: Linux 5.6.0-1-amd64 (x86_64)
ncurses: ncurses 6.2.20200212 (compiled with 6.2.20200212)
libidn: 1.33 (compiled with 1.33)
GPGme: 1.13.1-unknown
libnotmuch: 5.2.0
hcache backends: tokyocabinet

Configure options: --build=x86_64-linux-gnu --prefix=/usr {--includedir=${prefix}/include} {--mandir=${prefix}/share/man} {--infodir=${prefix}/share/info} --sysconfdir=/etc --localstatedir=/var --disable-silent-rules {--libdir=${prefix}/lib/x86_64-linux-gnu} {--libexecdir=${prefix}/lib/x86_64-linux-gnu} --disable-maintainer-mode --disable-dependency-tracking --mandir=/usr/share/man --libexecdir=/usr/libexec --with-mailpath=/var/mail --gpgme --lua --notmuch --with-ui --gnutls --gss --idn --mixmaster --sasl --tokyocabinet

Compilation CFLAGS: -g -O2 -fdebug-prefix-map=/build/neomutt-bCiSXv/neomutt-20191207+dfsg.1=. -fstack-protector-strong -Wformat -Werror=format-security -std=c99 -D_ALL_SOURCE=1 -D_GNU_SOURCE=1 -D__EXTENSIONS__ -I/usr/include -I/usr/include/lua5.3 -DNCURSES_WIDECHAR -isystem /usr/include/mit-krb5

Default options:
  +attach_headers_color +compose_to_sender +compress +cond_date +debug 
  +encrypt_to_self +forgotten_attachments +forwref +ifdef +imap +index_color 
  +initials +limit_current_thread +multiple_fcc +nested_if +new_mail +nntp +pop 
  +progress +quasi_delete +regcomp +reply_with_xorig +sensible_browser +sidebar 
  +skip_quoted +smtp +status_color +timeout +tls_sni +trash 

Kompileringsval:
  -autocrypt +bkgdset +color +curs_set +fcntl -flock -fmemopen +futimens 
  +getaddrinfo +gnutls +gpgme +gss +hcache -homespool +idn +inotify 
  -locales_hack +lua +meta +mixmaster +nls +notmuch -openssl +pgp +sasl +smime 
  -sqlite +start_color +sun_attachment +typeahead 
MAILPATH="/var/mail"
MIXMASTER="mixmaster"
PKGDATADIR="/usr/share/neomutt"
SENDMAIL="/usr/sbin/sendmail"
SYSCONFDIR="/etc"

To learn more about NeoMutt, visit: https://neomutt.org
If you find a bug in NeoMutt, please raise an issue at:
    https://github.com/neomutt/neomutt/issues
or send an email to: <neomutt-devel@neomutt.org>
```

## Extra Info

Konsole (KDE terminal) under Debian sid, using IMAP mail.

**Comment by flatcap** [comment]
We don't have a Norwegian translation, it turns out you're using the Swedish one (`sv`).

The problem comes from pairs of string that need to be translated together:
- Question: `(s)ign, (e)ncrypt`
- Option letters: `se`

I can fix _that_ question in _that_ language easily enough.
It may take me a little while to verify all the others (24 questions * 31 languages).

For future reference: I've added a log message, on startup, that will tell you which language is in use.

**Comment by Natureshadow** [comment]
Thanks for triaging the bug!

Actually, I have another system with the same neomutt config and version, but subtly different locale settings:

```
LANG=nb_NO.UTF-8
LANGUAGE=nb_NO:nb:no_NO:no:nn_NO:nn:da:sv:en
LC_CTYPE=nb_NO.UTF-8
LC_NUMERIC=nb_NO.UTF-8
LC_TIME=nb_NO.UTF-8
LC_COLLATE=C
LC_MONETARY=de_DE.UTF-8
LC_MESSAGES=nb_NO.UTF-8
LC_PAPER=de_DE.UTF-8
LC_NAME=nb_NO.UTF-8
LC_ADDRESS=de_DE.UTF-8
LC_TELEPHONE=nb_NO.UTF-8
LC_MEASUREMENT=nb_NO.UTF-8
LC_IDENTIFICATION=nb_NO.UTF-8
LC_ALL=
```

On that system, the message is displayed in Swedish, but the key binding for `clear` is `y` (from `ryd`, displayed correctly in the message). On the system mentioned in the original post, the message is displayed in English, and the binding is `b`. So a side question: Why does one system display a Swedish message, the other one in English, and why do they map different keys for the Swedish locale?

**Comment by flatcap** [comment]
> LANGUAGE=nb_NO:nb:no_NO:no:nn_NO:nn:**da**:sv:en

Ignoring all the Norwegian variants, this time the first match is `da` (Danish).

The Swedish question hasn't been translated, but the option letters **have**.
So you saw the default question (English), but NeoMutt's expecting the Swedish letters.

Both the Danish questions and letters have been translated,
so you see the right question and the right letters work.

Neither the Swedish nor Danish translations are maintained.
 
I _think_ I've got a clever solution to prevent the problem,
but working with the translations always takes so much time.

**Comment by flatcap** [comment]
I've 'fixed' the immediate problem, (buried) in f6631052a.
If any part of a _set_ of translated strings isn't translated, then none of them will be used.
(You'll still see all the other translations)

My 'clever' solution didn't quite pan out.
I'd hoped that I might be able to change one or two seldom-used key letters,
but once all the translations were factored in it turned out to be quite messy.

We have a longer-term solution to eliminate the option letter translations,
but that's low priority (unless we find a volunteer).

**Comment by Natureshadow** [comment]
Thanks a lot! If I actually can find out what to type now, I am all happy.

I will see what I can do concerning some translations.
